addons/event_sale/views/event_views.xml

```diff
@@ -26,4 +26,71 @@
         </field>
     </record>
 
+    <record id="view_event_sale_report_graph" model="ir.ui.view">
```

Comment:
Guidelines: views should be defined in their own file (event_sale_report_views.xml) so that each model as its own view, easier to find them.